docs/src/piccolo/getting_started/setup_cockroach.rst

```diff
@@ -0,0 +1,53 @@
+.. _setting_up_cockroach:
+
+##############
+Setup Cockroach
+##############
```

Comment:
RST is incredibly fussy - the number of hashes needs to match the title length. It just needs one more top and bottom.